packages/Algernon.package/ALGCore.class/class/showUnknownPlatformDialogSync.st

```diff
@@ -0,0 +1,10 @@
+accessing
```

Comment:
I don't think it belongs in this category

Reply:
Hm, yeah, not sure how it got in there :D